sale_exception_product_sale_manufactured_for/tests/test_sale_exception.py

```diff
@@ -1,10 +1,10 @@
 # Copyright 2021 Camptocamp SA
 # License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html)
 
-from odoo.tests import SavepointCase
+from odoo.tests import TransactionCase
```

Comment:
FYI, to avoid the `tracking_disable` context, you can use `BaseCommon` class 
`from odoo.addons.base.tests.common import BaseCommon`

Reply:
thanks for information